docs/installation/index.rst

```diff
@@ -3,17 +3,9 @@
 Installation and Configuration
 ------------------------------
 
-Interoperability
```

Comment:
did the interoperability get moved somewhere else, or just deleted?

Reply:
This got moved elsewhere IMHO.